apps/web/src/pages/dao/[network]/[token]/[tokenId].tsx

```diff
@@ -111,11 +112,19 @@ const TokenPage: NextPageWithLayout<TokenPageProps> = ({
       : baseSections
   }, [hasThreshold, collection])
 
-  // remove line breaks and formatting from og description
-  const cleanDesc = description.replace(/(\r\n|\n|\r|\t|\v|\f|\\n)/gm, '')
+  const ogDescription = useMemo(() => {
+    if (!description) return ''
+    const isMarkdown = isPossibleMarkdown(description)
 
-  const ogDescription =
-    cleanDesc.length > 111 ? `${cleanDesc.slice(0, 111)}...` : cleanDesc
+    if (isMarkdown) {
```

Comment:
Can we be a bit more explicit with the logic here -- perhaps in comments, why if the description is markdown to we not render the description?

Reply:
Yep! Will add a detailed explanation.